### ISSUE TITLE
Checkmark in field drop-down menus should probably be white

### ISSUE THREAD

**Issue by Jaqster** [microsoft/pxt-microbit#5789]
Right now it's black.  For color contrast, might be better to change it to white.
![image](https://github.com/user-attachments/assets/852644d7-f60f-4598-8df2-342740c6de7b)
![image](https://github.com/user-attachments/assets/700d57aa-7672-4544-a5df-6ef7b43f4add)

Also, there seems to be a lot of unnecessary padding on the right?
